mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/utbetaling/db/DelutbetalingQueries.kt

```diff
@@ -101,19 +100,13 @@ class DelutbetalingQueries(private val session: Session) {
         @Language("PostgreSQL")
         val query = """
             select
+                id,
```

Comment:
Jeg tror metoden `DelutbetalingQueries.get()` nå kunne tatt inn selve `id`'en i stedet for kombinasjonen av `utbetalingId + tilsagnId`? Delutbetalingen må vel være opprettet allerede, så id'en burde vel være tilgjengelig i frontend også ifm. operasjonene som gjøres der?